WalletWasabi.Fluent/App.axaml.cs

```diff
@@ -11,6 +11,7 @@
 using WalletWasabi.Fluent.Providers;
 using WalletWasabi.Fluent.ViewModels;
 using WalletWasabi.Fluent.Views;
+using Logger = WalletWasabi.Logging.Logger;
```

Comment:
Remove the unused usings except `using WalletWasabi.Logging;` and you can remove this line.

Reply:
ACK